node_io/lib/src/file.dart

```diff
@@ -69,8 +86,21 @@ class File extends FileSystemEntity implements io.File {
 
   @override
   Future<FileSystemEntity> delete({bool recursive: false}) {
-    // TODO: implement delete
-    throw new UnimplementedError();
+    if (recursive)
+      return new Future.error(new UnsupportedError(
+          'Recursive delete is not supported by Node API'));
```

Comment:
nit: I'd add `{}` around return statement here for readability.

Reply:
ok